packages/icons-react/README.md

```diff
@@ -35,11 +35,102 @@ respectively.
 To import using CommonJS, you can do the following:
 
 ```js
-const Add24 = require('@carbon/icons-react/lib/Add/24');
+const { Add24 } = require('@carbon/icons-react');
 ```
 
 _Note: if you would like to find the import path for an icon, you can reference
-our [icon preview](https://carbon-elements.netlify.com/icons/examples/preview/)_
+our
+[Icon Library](https://www.carbondesignsystem.com/guidelines/iconography/library)_
+
+### Icon fill
+
+All icons from the library support being styled by the `fill` property. You can
+change the color of an icon by passing in a custom class name that sets this
+property (preferred), or by passing in an inline style. For example:
+
+```css
+// CSS custom class name to set the fill of the icon to `rebeccapurple`
+svg.my-custom-class {
+  fill: rebeccapurple;
+}
+```
+
+```jsx
+import { Add16 } from '@carbon/icons-react';
+
+function MyComponent() {
+  return (
+    <button>
+      <Add16 aria-label="Add" className="my-custom-class" />
+    </button>
+  );
+}
+```
+
+#### Two-tone icons
+
+Certain icons in the library support two distinct fill colors. You can target
+the inner path by using the `[data-icon-path="inner-path"]` attribute selector.
+For example:
+
+```css
+// CSS custom class name to set the fill of the icon to `yellow`
+svg.outer-icon-fill {
+  fill: yellow;
+}
+
+// Use the `data-icon-path` attribute selector to target the inner path
+// where we want to set the fill to `black`
+svg.outer-icon-fill [data-icon-path='inner-path'] {
```

Comment:
Hmmm I wonder if we can’t just use classes here as well I think the outer svg should take a custom class but then they could target the inner classes we provide

Reply:
What would this look like?